quick-start.md

```diff
@@ -8,7 +8,16 @@
 Inspect the deployment file `deployments/quick-start/vault.yaml`. The deployment starts Vault in development mode with the root token
 set to `vault-root-token`. It is also started using `http`. In production, you should run Vault over `https`.
 
-Deploy: `kubectl apply -f deployments/quick-start/vault.yaml`
+- For kubernetes versions >= 1.6 deploy: 
```

Comment:
I think the RBAC config to apply to the `kubernetes-vault.yaml` deployment, since Vault itself does not talk to Kubernetes.